types/signals/index.d.ts

```diff
@@ -38,47 +39,56 @@ declare namespace signals {
 
         /**
          * If Signal is active and should broadcast events.
+         * IMPORTANT: Setting this property during a dispatch will only affect the next dispatch, if you want to stop the propagation of a signal use `halt()` instead.
          */
         active: boolean;
 
         /**
          * If Signal should keep record of previously dispatched parameters and automatically
          * execute listener during add()/addOnce() if Signal was already dispatched before.
+         *
+         * @type {boolean}
```

Comment:
This is redundant, already provided by the type declaration.

Reply:
Okay that's fixed 😉 
Do you have an answer for the question I asked in the description ?